search.py

```diff
@@ -10,7 +10,8 @@
 from collections import Counter
 from index import Posting, PostingList
 from encode import decode
-
+from nltk.corpus import wordnet
```

Comment:
👍 . nice TIL wordnet dont even need to make http request anymore

Reply:
Haha yep it doesnt. Its part of nltk so it can be used like a library

Reply:
Is the pr ok for merging?

Reply:
Go ahead

Reply:
ok merged!!